bin/ubuntu-core-initramfs

```diff
@@ -739,35 +740,28 @@ def create_efi(parser, args):
             except gzip.BadGzipFile:
                 pass
 
-    # kernel.efi sections will be aligned to the page size by llvm-objcopy
-    # (which is the default value for SectionAlignment in the PE header).
-    # --set-section-flags are such so added COFF sections have flags:
-    # CONTENTS, ALLOC, LOAD, READONLY, DATA
-    # (see llvm-objcopy man page for details)
-    # To see more details, run 'objdump --all-headers kernel.efi'
-    # (objdump gives a bit more of information than llvm-objdump in this case)
-    objcopy_cmd = [
-            "llvm-objcopy",
+    # TODO: add --splash
+    ukify_cmd = [
+        '/usr/lib/systemd/ukify', 'build',
+        '--linux', args.kernel,
+        '--initrd', args.initrd,
+        '--output', args.output,
+        '--os-release', '@{}'.format(args.os_release),
+    ]
+
+    if args.kernelver:
+        ukify_cmd += [
+            '--uname', args.kernelver,
+        ]
+    if args.stub:
+        ukify_cmd += [
+            '--stub', args.stub,
         ]
-    # TODO add .osrel
     if args.cmdline:
-        cmdline = tempfile.NamedTemporaryFile(mode='w')
-        cmdline.write('%s' % args.cmdline)
-        cmdline.flush()
-        objcopy_cmd += [
-            "--add-section", ".cmdline=%s" % cmdline.name,
-            "--set-section-flags", ".cmdline=readonly,data",
-            ]
-    # TODO add .splash
-    objcopy_cmd += [
-            "--add-section", ".linux=%s" % args.kernel,
-            "--set-section-flags", ".linux=readonly,data",
-            "--add-section", ".initrd=%s" % args.initrd,
-            "--set-section-flags", ".initrd=readonly,data",
-            args.stub,
-            args.output,
+        ukify_cmd += [
+            '--cmdline', args.cmdline,
```

Comment:
Will this be compatible with how we seal with FDE? I fear to expose something that might lead to misunderstandings on what is supported.

Reply:
This was already here. I have no idea who is using this.